dashboard/src/main/resources/js/dashboard.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function toggleSourceClassListVisibility(button) {
+  const classList = button.parentElement.nextElementSibling;
```

Comment:
Google JavaScript style uses `const`
https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
something like these for the symbols (not + and -) 

![image](https://user-images.githubusercontent.com/1005544/52817088-d33d6f80-3070-11e9-933a-b819261b032e.png)

Try Unicode character U+25B6 for a closed zippy icon and U+25BC for an open zippy icon.